### ISSUE TITLE
Longer term Enhancement relating to ECO and SCO 

### ISSUE THREAD

**Issue by KelleyDrake** [ddrake/Integrated_Farm_Budget_Tool#230]
Recall that you can scale county insurance from 80% to 120% and we have a button that allows for that called the selected payment factor.    ECO and SCO also have a selected payment factor that can range from 50% to 100%.    Just like for the area selected payment factor, this simply scales both the premium payments and the indemnity payments that would be paid or received at 100% at that factor.    I read an industry player's report today talking about how this could be a valuable feature for farmers particularly for this year.   Since Projected Prices are far lower in 2024 compared to 2023, the coverage that a typical policy provides is a lot lower because of the lower revenue guarantee.  Therefore, there is perhaps more interest in SCO and ECO overall.  This industry observer highighted the fact that ECO at the 90% or 95% level is expensive but also that you could have the high coverage triggering event but buy less of it.    For example you could buy a 95% ECO policy so that there is a 5% deductible where indemnity payment triggers -- but this policy at the 100% level is very expensive.   So the farmer might think about buying it (to get the high 95% coverage level, but also scale down the amount of that coverage to reduce the premium costs.   

Bottom line, I think we should eventually add two more selected payment factors (which will range from 50% to 100%) to the insurance selection screen.  So that it might look like this.   No rush on this at all but I think that would be a useful (and I think relatively simple) enhancement.

Selected Payment Factor SCO (50% to 100%)
Selected Payment Factor ECO (50% to 100%)
Selected Payment Factor County (80% to 120%)

**Comment by ddrake** [comment]
I agree, that should be easy to implement and explain to users.
